stdlib/tkinter/__init__.pyi

```diff
@@ -3331,9 +3331,32 @@ class PhotoImage(Image, _PhotoImageLike):
     def blank(self) -> None: ...
     def cget(self, option: str) -> str: ...
     def __getitem__(self, key: str) -> str: ...  # always string: image['height'] can be '0'
-    def copy(self) -> PhotoImage: ...
-    def zoom(self, x: int, y: int | Literal[""] = "") -> PhotoImage: ...
-    def subsample(self, x: int, y: int | Literal[""] = "") -> PhotoImage: ...
+    if sys.version_info >= (3, 13):
+        def copy(
+            self,
+            *,
+            from_coords: Iterable[int] | None = None,
+            zoom: int | tuple[int, int] | None = None,
+            subsample: int | tuple[int, int] | None = None,
+        ) -> PhotoImage: ...
+        def subsample(self, x: int, y: Literal[""] = "", *, from_coords: Iterable[int] | None = None) -> PhotoImage: ...
+        def zoom(self, x: int, y: Literal[""] = "", *, from_coords: Iterable[int] | None = None) -> PhotoImage: ...
+        def copy_replace(
+            self,
+            sourceImage: PhotoImage,
+            *,
+            from_coords: Iterable[int] | None = None,
+            to: Iterable[int] | None = None,
+            shrink: bool = False,
+            zoom: int | tuple[int, int] | None = None,
+            subsample: int | tuple[int, int] | None = None,
```

Comment:
This is a bit weird feature, but the source code checks for lists with `isinstance` so we might as well support it:

```suggestion
            zoom: int | tuple[int, int] | list[int] | None = None,
            subsample: int | tuple[int, int] | list[int] | None = None,
```

This will need to be copied to other methods with these args.

Reply:
I mostly did this to be a bit stricter, since it explicitly mentioned pairs or a single integer. But can also support this if you think it makes sense.

Reply:
https://github.com/python/typeshed/pull/12342/commits/3649ec8e10c57acb7b00dbae906f82f65c253450